Lib/fontTools/misc/psCharStrings.py

```diff
@@ -988,23 +988,36 @@ def compile(self, isCFF2=False):
 		encodeFixed = self.getFixedEncoder()
 		i = 0
 		end = len(program)
+		args = []
```

Comment:
Wouldn't something like
```python
if not isCFF:
    assert isinstance(program[-1], basestring), "T2CharString or Subr has items on the stack after last operator"
```
be good enough? Or a nice exception, of course.

Reply:
right. I'll simplify